src/pages/Scholarships.jsx

```diff
@@ -21,11 +24,51 @@ const useStyles = makeStyles((theme) => ({
 }));
 
 function ScholarshipsPage() {
+  const classes = useStyles();
   const [scholarships, setScholarships] = useState([]);
   const [loading, setLoading] = useState(true);
   const [error, setError] = useState();
-  const [sortField, setSortField] = useState('deadline');
-  const [sortDir, setSortDir] = useState('asc');
+  const history = useHistory();
+  const location = useLocation();
+
+  const validField = Boolean(
+    queryString.parse(location.search).sortField === 'deadline' ||
+      queryString.parse(location.search).sortField === 'amount.min' ||
+      queryString.parse(location.search).sortField === 'amount.max'
```

Comment:
I wonder if it'd be better to use `amount` instead of `amount.min`/`amount.max` in the `sortField`. We can determine whether to use min/max based on the `sortDir`.

Reply:
Well wouldnt it still display the amount.min and amount.max in the url?

Reply:
Alright I changed the verification for paras.sortField === 'amount' to use the `sortDir` IF it is valid, to set the sorting parameters appropriately, check and let me know if that is what you were thinking or if it should just go back to how it was.